### PR TITLE
Person - PersonProjectHistory - Created by id

### PR DESCRIPTION
**Pull request by kfaln**
Use proper CreatedBy user id (from config) and username when logging PersonProjectHistory.
Also when creating new users.

### CODE REVIEW

File: src/QueueReceiver.Infrastructure/Data/DbContextSettings.cs [hunk 1]
```diff
@@ -3,5 +3,8 @@
     public class DbContextSettings
```
Comment (TordJoranger) [inline src/QueueReceiver.Infrastructure/Data/DbContextSettings.cs]: I think we should move the  PersonProjectCreatedUsername to somewhere else. As it doesn't belong in this type or class or namespace anymore.

**Review by TordJoranger — APPROVED** [review]
With suggestions (and pending green tests)

File: tests/QueueReceiver.Core.UnitTests/Services/PersonProjectHistoryHelperTests.cs [hunk 2]
```diff
@@ -83,7 +83,7 @@ public void LogUnvoidProjects()
             var personProjectHistoryOperations = personProjectHistory.PersonProjectHistoryOperations.FirstOrDefault();
 
             Assert.IsTrue(personProjectHistory.PersonProjectHistoryOperations.Count == 1);
-            Assert.IsTrue(personProjectHistoryOperations.UpdatedByUser == PersonProjectHistoryConstants.UpdatedBy);
+            Assert.IsTrue(personProjectHistoryOperations.UpdatedByUser == "ACCESS_SYNC");
```
Comment (TordJoranger) [inline tests/QueueReceiver.Core.UnitTests/Services/PersonProjectHistoryHelperTests.cs]: Think we can just remove this line, as updatedByUser is no longer handled by this class. Should be tested in test class for PersonProjectHistoryRepository.

Comment (kfaln) [reply tests/QueueReceiver.Core.UnitTests/Services/PersonProjectHistoryHelperTests.cs]: DbContextSettings rewritten to PersonCreatedByCache

File: src/QueueReceiver.Infrastructure/Data/DbContextSettings.cs (hunk 1 above)
Comment (kfaln) [reply src/QueueReceiver.Infrastructure/Data/DbContextSettings.cs]: DbContextSettings rewritten to PersonCreatedByCache

**Review by TordJoranger — APPROVED** [review]

**Pull request merged**